src/StoreKeeper/WooCommerce/B2C/Commands/Guard.php

```diff
@@ -38,7 +43,7 @@ public function __construct(string $class)
         $this->file = $file;
     }
 
-    public function lock()
```

Comment:
this lock works differently than the MySqlLock.

MySqlLock will throw exception, this one will return false. They should work the same.

Reply:
All locks now throw boolean true then throw exceptions otherwise.